authors/apps/articles/models.py

```diff
@@ -129,3 +131,67 @@ class Favorite(models.Model):
         related_name='favorites')
     article_id = models.ForeignKey(
         Article, on_delete=models.CASCADE, related_name='favorites')
+
+
+class Tag(models.Model):
+    tag = models.CharField(max_length=230, db_index=True, unique=True)
```

Comment:
I feel like 230 characters I too long for a tag.

Reply:
Cool lemme reduce it to 100 characters. Thanks for the heads up @MandelaK